README.md

```diff
@@ -622,7 +622,7 @@ Other Style Guides
     });
     ```
 
-  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise use a `return` statement.
+  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise add the parentheses, braces, and use a `return` statement.
```

Comment:
@goatslacker i think you also wanted this to be explicit about whether the parens around a single arrow function argument were required, prohibited, or either, and when - if the intention is that you must always use parens around arrow function args that have a body with braces, i think that should be explicitly stated.